conmon-rs/server/src/child_reaper.rs

```diff
@@ -143,7 +143,7 @@ impl ChildReaper {
 
     pub fn kill_grandchildren(&self, s: Signal) -> Result<()> {
         for (_, grandchild) in self.grandchildren.lock().unwrap().iter() {
-            debug!("{}: killing grandchild", grandchild.pid);
+            debug!(grandchild.pid, "killing grandchild");
```

Comment:
I think this work got mixed with the toolchain bump, can we separate?

Reply:
I would even prefer we just merge this stuff with the original commit, as the git tree will look as though span support was added before this anyway

Reply:
fixed that up.